.scalafmt.config

```
@@ -0,0 +1,20 @@
+maxColumn = 120
+align = most
+continuationIndent.defnSite = 2
+assumeStandardLibraryStripMargin = true
+docstrings = ScalaDoc
+lineEndings = preserve
+includeCurlyBraceInSelectChains = false
+danglingParentheses = true
+
+align.tokens = [
```

I think that this is resulting in *only* `:` being aligned. I think the intent was that `:` would *also* be aligned. Concretely, this seems to be causing `=>` to not be aligned which was one of the things I think we wanted?

```suggestion
align.tokens.add = [
```

---

I like it. I will make the change. The weird thing I checked a few => and thought they were aligned.

---

The change was made and firrtl had been reformatted again and is pushed